src/components/encounter/EncounterList.tsx

```diff
@@ -19,7 +19,7 @@ const EncounterEntry = ({ encounter }: { encounter: Encounter }) => {
         <RegularText>{momentFormat(encounter.createdAt, 'HH:mm')}</RegularText>
       </Wrapper>
       <Wrapper>
-        <RegularText>{`${texts.encounter.id}: ${encounter.encounterId}`}</RegularText>
+        <RegularText selectable>{`${texts.encounter.id}: ${encounter.encounterId}`}</RegularText>
```

Comment:
> encounter and user ids are now selectable

why exactly? what can this be useful for?

Reply:
the id is a uuid and needs to be handled by people for verification/getting info about an encounter if needed.
if they are not selectable, they need to be copied correctly by hand.

Reply:
so this can be handy if someone people uses their phones for verification, right? i thought of people will mostly verify on desktops (from work?).

Reply:
it is useful for getting the users information from the phone to the pc from the administator